aten/src/ATen/native/cpu/DistributionKernels.cpp

```diff
@@ -149,13 +149,13 @@ void exponential_kernel(TensorIteratorBase &iter, double lambda, c10::optional<G
             vslNewStream(&stream, VSL_BRNG_MCG31, seed);
             vslSkipAheadStream(stream, begin);
             vdRngExponential(VSL_RNG_METHOD_EXPONENTIAL_ICDF, stream, len,
-              (double *)(sample_ptr + begin), 0, 1./lambda);
+              (double *)(sample_ptr + begin), 1.4013e-45, 1./lambda);
```

Comment:
Please don't hardcode values, use numeric_limits::denorm_min instead https://en.cppreference.com/w/cpp/types/numeric_limits/denorm_min. Do you want denorm_min here or just min though? Some systems might compile without denorm support.